geoapps/inversion/line_sweep/driver.py

```diff
@@ -39,20 +39,21 @@ def run(self):  # pylint: disable=W0221
 
     def setup_params(self):
         h5_file_path = Path(self.workspace.h5file).resolve()
-        ui_json_path = h5_file_path.with_suffix(".ui.json")
-        if not ui_json_path.is_file():
+        ui_json_path = h5_file_path.stem.replace(".ui", "") + ".ui.json"
```

Comment:
I do not understand when this was getting a wrong output

`with_suffix` seems to replace as expected. See example below:
![image](https://github.com/MiraGeoscience/geoapps/assets/24550538/d0384360-9db6-493e-b4d3-ce85fcae1295)

Reply:
ho, ok, it does stem, AND replace, I missed that. thus removing 2 extensions.
Could it be then `f"{h5_file_path.stem}.json"` ?

Reply:
guess so yes